install/ui/test/data/ipa_init.json

```diff
@@ -315,6 +315,18 @@
                             "view_certificate": "Certificate for ${entity} ${primary_key}",
                             "view_certificate_btn": "View Certificate"
                         },
+                        "certmap": {
```

Comment:
If I'm correct, some of your other PRs no longer update this file. We've talk to remove the offline mode. So what about removing it? Or is this still used for unit tests?